test/src/lib.rs

```diff
@@ -13,6 +13,10 @@ mod nomod {
     pub mod inner;
 }
 
+// This non-existent module should not be parsed
+#[cfg(feature = "non_existent")]
```

Comment:
Maybe there could be a more advanced test with a module containing invalid code for example.

Reply:
Well, can be, but this code already doesn't compile if you remove the `[#cfg(...)]` so I thought it was unnecessary to go further

Reply:
sure, but ideally, there should be as many tests as possible to cover as many case as possible.  For example, we could also imagine to ignore non-existant module. And so this test would still pass, and we would not have a test anymore for this feature.
Or we could imagine a mode where we parse every .rs files in the subdirectory no matter which module.

For example, does it work with 

```
#[cfg(feature = "non_existent")]
mod foobar {
  cpp!{{ compile error!  }}
}
```

Reply:
You're right, for some reason I thought you meant invalid Rust code in your initial comment. I'll update the tests